desktop/components/has_seen/dismisser.coffee

```diff
@@ -1,20 +1,24 @@
 _ = require 'underscore'
 Cookies = require '../cookies/index.coffee'
 
+# Get around inabilty to stub `document`
+domain = document.domain if typeof document isnt 'undefined'
+
 module.exports = class Dismisser
   defaults:
     limit: 5
     expires: 31536000 # 1 Year
+    domain: domain
```

Comment:
There are other values that get passed in: https://github.com/damassi/force/blame/master/desktop/components/has_seen/dismisser.coffee#L17